### PR TITLE
More instructions to open issues

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

**Comment by willingc** [comment]
@Carreau :smile: I am making some edits and will send them to your repo as a PR.

**Comment by Carreau** [comment]
> @Carreau  I am making some edits and will send them to your repo as a PR.

You can push directly to my branch too if you like.

**Comment by Carreau** [comment]
@willingc merged your PR.

**Comment by willingc** [comment]
Thanks @Carreau. Once the Spanish phrase is translated, I think this is ready to merge (unless others have suggestions).

**Comment by Carreau** [comment]
ping @damianavila for translation help !

File: README.md [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
+  "Submit new Issue". Congratulations! You submitted your new issue! We'll try
+  to get back to you shortly. Tip: If you provide helpful info (such as
+  operating system, versions, etc.), it's easier for us to understand your
+  question :exclamation:
+
+> Je ne parles pas Anglais (I don't speak English.) :question:
+
++ Pas de problèmes, on a des francophones (No problem, we speak French too.) :exclamation:
+
+> No habla ingles :question:
```
Comment (damianavila) [inline README.md]: Pero no hablo en Inglés

File: README.md [hunk 2]
```diff
@@ -1,9 +1,41 @@
...
+
++ TODO (find someone who speak spanish to fill that in) :exclamation:
```
Comment (damianavila) [inline README.md]: No te hagas problema, tambien hablamos en Español

**Comment by damianavila** [comment]
>ping @damianavila for translation help !

Added comments with the Spanish part...

**Comment by rgbkrk** [comment]
Love this

**Comment by Carreau** [comment]
> Added comments with the Spanish part...

`git commit --author='damianavila <damianavila@gmail.com>'` done.

**Comment by damianavila** [comment]
> git commit --author='damianavila <damianavila@gmail.com>' done.

Thanks... sorry to not commit by myself... I was at the phone at that moment :wink:

**Comment by Carreau** [comment]
> Thanks... sorry to not commit by myself... I was at the phone at that moment

No worries ! I still wanted to give you credits !

Thanks !

**Pull request merged**